common/traits/wc_genetic_traits.txt

```diff
@@ -224,4 +226,51 @@ magic_good_noninheritable = {
 			desc = trait_magic_good_noninheritable_character_desc
 		}
 	}
-}
\ No newline at end of file
+}
+
+dire = {
+	index = 2508
+	opposites = {
+		weak
+		physique_bad
+		dwarf
+		giant
+	}
+	
+	potential = {
+		OR = {
+			dire_possible_trigger = yes
+			mother = { dire_possible_trigger = yes }
+			real_father = { dire_possible_trigger = yes }
+		}
+	}
+	
+	birth = 0.5
+	random_creation = 0.5
+
+	prowess = 13
+	dread_baseline_add = 15
+	martial = 8
+	diplomacy = -7
+	health = -1
+	enemy_hard_casualty_modifier = 0.15
```

Comment:
This **GIANT** boost to martial makes this trait like the best martial education... plus `enemy_hard_casualty_modifier`. I don't think dire trolls are born geniuses of strategy. Moreover, the ones on Zandalar are pretty dummies. I would limit the bonus to Prowess and Dread. Prowess is enough to make them good commanders that don't die and great knights to make the army stronger.

Reply:
Changed this in e3793d283bdb6dec711901887bee2473c807097c.

Removed everything except Prowess and Dread attributes, although I did increase Prowess from 13 to 15 to make up for the removed attributes. Updated the trait on the [balance sheet](https://docs.google.com/spreadsheets/d/18zS-FAte6nsejx7rpzSmVuioRXps3iWSvSkx_xRJSWI/edit#gid=1768507286).